projects/core/src/lib/project/project.service.ts

```diff
@@ -863,8 +865,7 @@ export class ProjectService {
 
         return combineLatest(observables).pipe(
             skip(1),
-            take(1),
-            map(() => {}),
+            map((a) => a[1].resolution),
```

Comment:
What is **a** and why do we take the resolution of the "second a"?

Reply:
857ab2e12b8cf1b629860ce691677f01650c8d14 better? please make a suggestion instead of a rhetorical question then.